templates/helm/orchestration-workflows/clinvar/values.schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "env": { "type": "string", "enum": ["dev"] },
+    "properties": {
+      "enable": { "type": "boolean" },
+      "schedule": { "type": "string" }
+    }
+  },
+  "required": ["env", "enable"]
```

Comment:
`enable` isn't in the right scope to be required here, I think. Shouldn't `enable` and `schedule` be pulled up to be on the same level as `env`? If not, they're not correctly nested as sub-properties under the `env` property.